### PR TITLE
Screen Reader Changes for Address Blocks

### PR DESCRIPTION
**Pull request by jage9**
* Added labels for all address fields
* Added a heading role for each section (heading level 2)
* Hid the duplicate graphic from  screen readers as it is the same text as the title
* Added a group role and label for each section

Reference to issue #195

### CODE REVIEW

**Comment by jage9** [comment]
Oops, there's one small error with the country field. Troubleshooting that now.

File: YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php [hunk 1]
```diff
@@ -18,85 +18,85 @@
...
-<table class="table">
-    <tr>
-        <td><?php echo zen_image(DIR_WS_IMAGES . $address_icon, $address_label); ?></td>
-        <td class="eo-label"><?php echo $address_label; ?></td>
-    </tr>
+<div role="group" aria-labelledby="sr-<?php echo $address_name; ?>">
+    <table class="table">
+        <tr>
+            <td aria-hidden="true"><?php echo zen_image(DIR_WS_IMAGES . $address_icon, $address_label); ?></td>
+            <td class="eo-label" id="sr-<?php echo $address_name; ?>" role="heading" aria-level="2"><?php echo $address_label; ?></td>
```
Comment (lat9) [inline YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php:25]: For my learning process, what does the aria-level="2" signify?

File: YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php [hunk 2]
```diff
@@ -18,85 +18,85 @@
...
+        </tr>
+        <tr>
+            <td class="eo-label"><label for="update_<?php echo $address_name; ?>_name"><?php echo ENTRY_CUSTOMER_NAME; ?></label>:&nbsp;</td>
```
Comment (lat9) [inline YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php:28]: Also for 'learning', it's important for screen-readers to ensure that there's a `label` tag associated with each input?

File: YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php [hunk 3]
```diff
@@ -18,85 +18,85 @@
...
+    <?php
+        }
     }
-}
-?>
-</table>
-
+    ?>
+    </table>
+</div>
```
Comment (lat9) [inline YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php:102]: Please make sure that there's an empty line at the end.

**Review by lat9 — CHANGES_REQUESTED** [review]
Thanks, as always, @jage9.  This is (as noted) a learning experience for me and I truly appreciate the time you've spent!

**Comment by jage9** [comment]
Happy to help.

Label is one way to add an implicit title for a form field, and is often prefered, as the browser does not need to guess as to which text is the actual label. This is the method used in much of the responsive_classic template, which is generally compliant with accessibility standards.

role="heading" aria-level="2"
is the same as adding an H2 heading to the page. I used this method as to not disturb current visual formatting. One could also do an H2 with a CSS class to achieve formatting.

Let me know if you have any more questions.

File: YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php (hunk 3 above)
Comment (jage9) [reply YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php]: Done.

**Review by lat9 — APPROVED** [review]

**Pull request merged**